src/alpaka/Makefile

```diff
@@ -38,8 +38,7 @@ endif
 ifdef ROCM_BASE
 MY_CXXFLAGS += -DALPAKA_ACC_GPU_HIP_PRESENT -DALPAKA_ACC_GPU_HIP_ONLY_MODE
 endif
-MY_LDFLAGS := -ldl -Wl,-rpath,$(LIB_DIR)/$(TARGET_NAME)
-LIB_LDFLAGS := -L$(LIB_DIR)/$(TARGET_NAME)
+MY_LDFLAGS := -ldl -Wl,-rpath,$(LIB_DIR)/$(TARGET_NAME) -L$(LIB_DIR)/$(TARGET_NAME)
```

Comment:
I suppose strictly speaking `-ldl` is needed only for the `$(TARGET)`, but doesn't really harm for the shared libraries.

Reply:
I guess I can move it explicitly to the `$(TARGET)` line - though of course it doesn't harm elsewhere.

Reply:
I don't have preferences to either way.

Reply:
... we already have `-ldl` in the global `$(LDFLAGS)`, so I will just remove it from the `$(MY_LDFLAGS)`